tgui/packages/tgui/interfaces/CrewManifest.js

```diff
@@ -0,0 +1,89 @@
+import { classes } from 'common/react';
+import { useBackend } from '../backend';
+import { Icon, Section, Table, Tooltip } from '../components';
+import { Window } from '../layouts';
+
+const commandJobs = [
+  'Head of Personnel',
+  'Head of Security',
+  'Chief Engineer',
+  'Research Director',
+  'Chief Medical Officer',
```

Comment:
I reaaalllly don't like this. It should use ui_static_data and pass it from DM as JOB_NAME defines or the command_positions global